hr_holidays_public/__manifest__.py

```diff
@@ -1,14 +1,16 @@
 # Copyright 2015 2011,2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# Copyright 2019 Camptocamp
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
     'name': 'HR Holidays Public',
-    'version': '12.0.1.0.1',
+    'version': '13.0.1.0.0',
     'license': 'AGPL-3',
     'category': 'Human Resources',
     'author': "Michael Telahun Makonnen, "
               "Tecnativa, "
               "Fekete Mihai (Forest and Biomass Services Romania),"
+              "Camptocamp,"
```

Comment:
Not author, better add yourself as contributor in the README

Reply:
Yes, this migration doesn't grant your company as co-author